aws/resource_aws_autoscaling_group.go

```diff
@@ -242,6 +242,11 @@ func resourceAwsAutoscalingGroup() *schema.Resource {
 				Elem:          &schema.Schema{Type: schema.TypeMap},
 				ConflictsWith: []string{"tag"},
 			},
+
+			"service_linked_role_arn": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
I believe you'll need to add `Computed: true,` for this new attribute as well since I'm getting all test failures with:
```
=== RUN   TestAccAWSAutoScalingGroup_ALB_TargetGroups
--- FAIL: TestAccAWSAutoScalingGroup_ALB_TargetGroups (78.41s)
    testing.go:518: Step 0 error: After applying this step, the plan was not empty:
        
        DIFF:
        
        UPDATE: aws_autoscaling_group.bar
          service_linked_role_arn: "arn:aws:iam::*******:role/aws-service-role/autoscaling.amazonaws.com/AWSServiceRoleForAutoScaling" => ""

=== RUN   TestAccAWSAutoScalingGroup_ALB_TargetGroups_ELBCapacity
--- FAIL: TestAccAWSAutoScalingGroup_ALB_TargetGroups_ELBCapacity (336.79s)
    testing.go:518: Step 0 error: After applying this step, the plan was not empty:
        
        DIFF:
        
        UPDATE: aws_autoscaling_group.bar
          service_linked_role_arn: "arn:aws:iam::*******:role/aws-service-role/autoscaling.amazonaws.com/AWSServiceRoleForAutoScaling" => ""

...
```

Reply:
Hmm, interesting the tests pass for me both with and without `Computed: true`... I'll make the amendment in the moment.

Reply:
It might be related to if an AWS account has already had the service linked role automatically created by AWS. Running the acceptance tests again!

Reply:
Ah ha, here it is:

> For existing EC2 Auto Scaling customers, EC2 Auto Scaling will add a default EC2 Auto Scaling service-linked role in your account and your existing Auto Scaling groups will be updated to use this role over the next few weeks.

https://aws.amazon.com/about-aws/whats-new/2018/02/amazon-ec2-auto-scaling-adds-support-for-service-linked-roles/